### PR TITLE
Added Lingzhi and Ambrosia Vulgaris

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Botany power creep time.

**Screenshots**

harvest
Note: The examples given are at 100 potency.

lingzhi
100 potency lingzhi yield
ambrosia vulgaris
100 potency ambrosia vulgaris yield

**Changelog**

:cl: Rane
- add: Added lingzhi and ambrosia vulgaris, two more potent but slightly dangerous medicinal plants.
- tweak: Instead of kelotane and nutriment, aloe now yields dermaline and... wait for it... aloe!

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -621,3 +621,51 @@
...
+- type: entity
+  name: ambrosia vulgaris
+  parent: FoodProduceBase
+  id: FoodAmbrosiaVulgaris
+  description: A medicinal plant. May make you feel a little funny.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 22
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:656]: 24

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml [hunk 2]
```diff
@@ -225,3 +225,25 @@
...
+    - type: Seed
+      seed: lingzhi
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/lingzhi.rsi
+
+- type: entity
+  parent: SeedBase
+  name: packet of ambrosia vulgaris seeds
+  description: "A medicinal plant for the common folk."
+  id: AmbrosiaVSeeds ##The megaseed servitor won't load these if they have a fuller name and I have no idea why
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml:244]: ..what

Comment (mirrorcult) [reply Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml:244]: did the longer name you were trying to use already exist?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml (hunk 2 above)
Comment (Elijahrane) [reply Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml:244]: No

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**